server/push/fcm/push_fcm.go

```diff
@@ -175,6 +236,19 @@ func sendNotifications(rcpt *push.Receipt, config *configType) {
 		return
 	}
 
+	var titlelc, title, bodylc, body, icon, color string
+	if config.Android.Enabled {
+		titlelc = config.Android.getTitleLocKey(rcpt.Payload.What)
+		title = config.Android.getTitle(rcpt.Payload.What)
+		bodylc = config.Android.getBodyLocKey(rcpt.Payload.What)
+		body = config.Android.getBody(rcpt.Payload.What)
+		if body == "$content" {
+			body = data["content"]
```

Comment:
Should we also check if `body` is empty here?

Reply:
Maybe it's intentionally blank.